fala/apps/adviser/laa_laa_paginator.py

```diff
@@ -0,0 +1,47 @@
+class LaaLaaPaginator(object):
```

Comment:
What does this do that we needed to write our own instead of using Django's paginator like we discussed?

Reply:
Django's paginator doesn't work here. The items are already paginated - Django assumes you have an array to be paginated